srkServer/srkproto/srkproto.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package srkproto;
+
+service FunctionService {
+    
+
+    // ByteTransfer must represent a tar file with a single top-level directory.
+    // The context should include metadata with the following fields:
```

Comment:
Should metadata be exposed as part of the RPC protocol? Maybe as a map?

Reply:
The metadata is necessary because protobuf does not allow mixed scalar and stream arguments. This would mean that bulk byte transfers would have to carry a copy of the metadata everytime. As far as I know, this is the expected use-case for metadata.

Reply:
Ok, seems fine then.